### PR TITLE
Admin user mgmt skb

### PR DESCRIPTION
**Pull request by stellakunzang**
Add notes for refactor to routes
Add link to view for admin to ship order
Remove edit my profile from admin view of user profile
Add conditional for cancelling order - only if not shipped or cancelled
Update order status to shipped in admin order controller
Update admin merchants index to show city and state
Fix test by logging in a user
Update view for admin user index

### CODE REVIEW

File: app/views/admins/show.html.erb [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <p>User: <%= link_to "#{order.user.name}", "/admin/profile/#{order.user.id}" %></p>
     <p>Order Id: <%= "#{order.id}" %></p>
     <p>Date Order Placed: <%=  "#{order.created_at.strftime('%m/%d/%Y')}" %><p>
+    <p><%= link_to "Ship Order", "/admin/orders/#{order.id}", method: :patch %></p>
```
Comment (stellakunzang) [inline app/views/admins/show.html.erb:9]: Yo! We can ship an order now!

File: config/routes.rb [hunk 2]
```diff
@@ -12,10 +12,11 @@
   namespace :admin do
     get '/merchants/:merchant_id', to: "merchants#show"
     get '/merchants', to: "merchants#index"
-    get "/merchants/disable/:id", to: "merchants#disable"
-    get "/merchants/enable/:id", to: "merchants#enable"
+    get "/merchants/disable/:id", to: "merchants#disable" # merchant update (or put in a new status controller for merchant and updating there)
+    get "/merchants/enable/:id", to: "merchants#enable" # merchant update (or put in a new status controller for merchant and updating there)
```
Comment (stellakunzang) [inline config/routes.rb:16]: notes for refactoring routes to be more restful

File: app/views/admins/show.html.erb (hunk 1 above)
Comment (muydanny) [reply app/views/admins/show.html.erb:9]: !!!!

**Pull request merged**